spec/controllers/tag_controller_spec.rb

```diff
@@ -0,0 +1,35 @@
+require "spec_helper"
+
+describe TagsController do
+  before { allow(controller).to receive(:require_logged_in_admin) }
+
+  context 'create' do
+    it 'creates new tags' do
+      post :create, params: { tag: { tag: 'my tag' } }
```

Comment:
Great to have a couple specs. Could you change this to `'mytag'`? I'm pretty sure the site assumes tags to be single words. Some of the models only edited by the admin in console are missing validations (mostly presence of references).